### ISSUE TITLE
why the size of image shrinks, after using colmap to reconstruct ?

### ISSUE THREAD

**Issue by TSlus** [colmap/colmap#3057]
hi,

I use colmap to reconstruct,
the size of input image is: 5472 x 3648
But the size of image in the output of colmap is : 5466 x 3643

I used the colmap commands like:
~~~bash
colmap feature_extractor
colmap exhaustive_matcher
colmap mapper
colmap image_undistorter
~~~
Is the command `colmap image_undistorter` will change the image size ?

**Comment by B1ueber2y** [comment]
Yes. ImageUndistorter will change the image size to keep the same field of view (FOV) as the input image. If you really want to keep the same image size and potentially sacrifice FOV or get black regions, you could do the following (not recommended):

```
colmap image_undistorter --min_scale 1.0 --max_scale 1.0
```

**Comment by ahojnnes** [comment]
Closing: image_undistorter changes size to preserve FOV; explained.
